src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -437,13 +438,63 @@ object Helpers extends SparkSessionWrapper {
     s"SHRED COMPLETE: ${target.tableFullName}"
   }
 
+
+  /**
+   * Execute a parallelized streaming to follow the instructions provided through CloneDetail class
+   *
+   * @param cloneDetails details required to execute the parallelized clone
+   * @return
+   */
+  private[overwatch] def snapStream(cloneDetails: Seq[CloneDetail],snapshotRootPath: String): Unit = {
+
+    val cloneDetailsPar = cloneDetails.par
+    val taskSupport = new ForkJoinTaskSupport(new ForkJoinPool(parallelism))
+    cloneDetailsPar.tasksupport = taskSupport
+
+    logger.log(Level.INFO, "Streaming START:")
+    val cloneReport = cloneDetailsPar.map(cloneSpec => {
+      try {
+        val rawStreamingDF = spark.readStream.format("delta").load(s"${cloneSpec.source}")
+        val sourceName = s"${cloneSpec.source}".split("/").takeRight(1).head
+        val checkPointLocation = if (snapshotRootPath.takeRight(1) == "/") s"${snapshotRootPath}checkpoint/${sourceName}" else s"${snapshotRootPath}/checkpoint/${sourceName}"
+        val cloneReportPath = if (snapshotRootPath.takeRight(1) == "/") s"${snapshotRootPath}report/" else s"${snapshotRootPath}/clone_report/"
+        println("Source Path is",s"${cloneSpec.source}")
+        println("Target Path is",s"${cloneSpec.target}")
+        println("report path is ",cloneReportPath)
+        rawStreamingDF
+          .writeStream
+          .format("delta")
+          .trigger(Trigger.Once())
+          .option("checkpointLocation", checkPointLocation)
+          .option("path", s"${cloneSpec.target}")
+          .start()
+          .awaitTermination()
+        logger.log(Level.INFO, s"Streaming COMPLETE: ${cloneSpec.source} --> ${cloneSpec.target}")
+        CloneReport(cloneSpec, s"Streaming For: ${cloneSpec.source} --> ${cloneSpec.target}", "SUCCESS")
+      } catch {
+        case e: Throwable if (e.getMessage.contains("is after the latest commit timestamp of")) => {
+          val msg = s"SUCCESS WITH WARNINGS: The timestamp provided, ${cloneSpec.asOfTS.get} " +
```

Comment:
Use the `appendStackStrace` method shown in other comment to build this message with the full stack trace as well